### ISSUE TITLE
Cannot generate models

### ISSUE THREAD

**Issue by saggafarsyad** [phalcon/phalcon-devtools#288]
I can't generate models in phalcon-devtools 1.2.5. 

Using webtools.php returns error:
> Adapter is not supported

Using command-line it returns error like this:
> Error: Unable to write to './public//var/www/project/app/config/../../app/models/user.php

I'm running phalcon on vagrant, using phalconbox which I download from http://phalconphp.com/en/download/vagrant

**Comment by 7thcubic** [comment]
i think it is something to do with your database adapter. show config?

**Comment by rianorie** [comment]
that path looks messed up as well..

**Comment by 7thcubic** [comment]
or is it because of chmod?

**Comment by jbhamilton** [comment]
I just ran into this same issue. I fixed it by commenting out these lines in the file: 
> scripts/Phalcon/Commands/Builtin/AllModels.php

```
             //if (file_exists($path.'public')) {                                                                        
             //    $modelsDir = 'public/'.$config->application->modelsDir;                                               
             //} else {                                                                                                  
                 $modelsDir = $config->application->modelsDir;                                                           
             //}     
```
Not sure what the intent was with that if statement.

**Comment by inksable** [comment]
Same problem for me resolved ... Thanks jbhamilton

**Comment by Bati9999** [comment]
same issue with scripts/Phalcon/Commands/Builtin/Models.php;  script generate a mess path

**Comment by sergeyklay** [comment]
Fixed in 2.0.x
